helm/prometheus-meta-operator/templates/prometheus-rules/up.all.rules.yml

```diff
@@ -56,6 +56,22 @@ spec:
         severity: page
         team: ludacris
         topic: observability
+    - alert: KubeStateMetricsMissing
+      annotations:
+        description: '{{`KubeStateMetrics ({{ $labels.instance }}) is missing.`}}'
+        opsrecipe: kube-state-metrics-down/
+      expr: absent(up{app="app_name"} == 1)
```

Comment:
I think the app name is not correct here :D

Reply:
Argh - fix incoming